### PR TITLE
Pundit part 3 with companies controller

### PR DESCRIPTION
**Pull request by Stefanode**

### CODE REVIEW

File: app/controllers/companies_controller.rb [hunk 1]
```diff
@@ -14,7 +16,6 @@ def index
   end
 
   def show
-    @company = Company.find(params[:id])
```
Comment (pimmesz) [inline app/controllers/companies_controller.rb]: Why leave this out?

Comment (pimmesz) [reply app/controllers/companies_controller.rb]: Isnt it logical to also allow for a specific company to be found? It's also left out in the skip_before_action so it should be in here as well right?

Comment (Stefanode) [reply app/controllers/companies_controller.rb]: @pimmesz I like it dry 😏

**Comment by Stefanode** [comment]
[Pundit part 3](https://trello.com/c/T9tvzdPu/127-pundit-part-3)

**Comment by Stefanode** [comment]
[#25 As a user, I can add a company](https://trello.com/c/jzsTEISU/94-25-as-a-user-i-can-add-a-company)

**Comment by pimmesz** [comment]
[As a user I can view an organization](https://trello.com/c/HY934jBI/110-as-a-user-i-can-view-an-organization)

**Pull request merged**